integration_tests/tests/end_to_end.rs

```diff
@@ -245,6 +246,11 @@ async fn e2e_test_wrapper<T: Future<Output = ()>>(test: T) {
     let api_pool = db_pool.clone();
     let ingestion_pool = db_pool.clone();
 
+    let rove_connector = Connector {
+        pool: db_pool.clone(),
+    };
+    let qc_pipelines = load_pipelines("mock_qc_pipelines/fresh").expect("failed to load pipelines");
```

Comment:
Why do we need a separate file here? Can't we simply use `qc_pipelines/fresh/TA_PT1H.toml`? We probably need a helper method to load a single file though.

Reply:
My thought behind this was that in the real pipelines, params/steps might change such that the QC output changes, and we eventually want to be checking QC output in the integration tests. That could result in param/step changes breaking CI in ways they shouldn't.

While this isn't true right now (the mock and real pipelines are currently identical), it will be true in the future. Additionally, people updating pipeline definitions will eventually use CI to verify their pipelines are valid. I don't want them to be thrown off by their changes causing unrelated CI failures.

Reply:
I'm a bit confused by the last two sentences, but how can they verify that the changes are valid if they are not tested? Maybe I'm just sleepy :sweat_smile:

Reply:
Sorry, I was unclear. In the future, there will be a new test that loads all the real pipelines, checks they deserialise correctly, and perhaps also provides some linting